### DISCUSSION TITLE
Expo not working on examples/with-react-native-web on Mac

### DISCUSSION THREAD

**Discussion by lancejpollard** [vercel/turborepo#5219] [Help] [answered]
I just went through and npx'd this https://github.com/vercel/turbo/tree/main/examples/with-react-native-web

I have modified some of the folders and such, but not the script commands in package.jsons or anything [here](https://github.com/tunebond/term.land).

When I run `pnpm dev` at the top of the project, it says (amongst other things):

```
dock:dev:
dock:dev: > dock@1.0.0 start /Users/lancepollard/start/tunebond/deck/term.land/site/dock
dock:dev: > expo start
dock:dev:
dock:dev: This command requires Expo CLI.
dock:dev: Do you want to install it globally [Y/n]?
```

I can't press y/n in this logging context though, so it's kind of useless.

Where/how do I install expo for this repo?

I tried running this before as well:

```
brew update
brew install watchman
npm install -g pnpm
npx expo start
```

But the last command fails:

```
$ npx expo start
Starting project at /Users/lancepollard/start/tunebond/deck/term.land
ConfigError: Cannot determine which native SDK version your project uses because the module `expo` is not installed. Please install it with `yarn add expo` and try again.
```

Do I need to `cd` into the native app (in `site/dock` in my repo) and run `pnpm start`? I am new to turbo so I am unsure what it should automatically do yet.

If I do that I get:

```
$ cd site/dock
$ pnpm start

> dock@1.0.0 start /Users/lancepollard/start/tunebond/deck/term.land/site/dock
> expo start

This command requires Expo CLI.
Do you want to install it globally [Y/n]? y
Installing the package 'expo-cli'...
Expo CLI installed. You can run `expo --help` for instructions.
WARNING: The legacy expo-cli does not support Node +17. Migrate to the new local Expo CLI: https://blog.expo.dev/the-new-expo-cli-f4250d8e3421.
Starting project at /Users/lancepollard/start/tunebond/deck/term.land/site/dock
Some dependencies are incompatible with the installed expo package version:
 - react - expected version: 17.0.2 - actual version installed: 18.2.0
 - react-dom - expected version: 17.0.2 - actual version installed: 18.2.0
 - react-native-web - expected version: 0.17.7 - actual version installed: 0.18.9
Your project may not work correctly until you install the correct versions of the packages.
To install the correct versions of these packages, please run: expo doctor --fix-dependencies,
or install individual packages by running expo install [package-name ...]
```

So that seems wrong, so I removed that with `npm uninstall -g expo-cli`, but their [blog link](https://blog.expo.dev/the-new-expo-cli-f4250d8e3421) doesn't show how to get this working.

Thanks for your help.

**Comment by fahimmehraj** [comment]
I am also having this problem

**Comment by anthonyshew** [comment, marked as answer]
We've updated our examples recently so *hopefully* this is stale. Please open an Issue if you are still having problems. 🙏
